interfaces/products.ts

```diff
@@ -1,8 +1,17 @@
 export interface Product {
   id: number;
-  image: string;
+  image: Image;
   name: string;
+  content: Product;
+  category: Category;
   origin: string;
   price: number;
   quantity?: number;
 }
+interface Image {
+  __typename: string;
```

Comment:
Vc não tá buscando essa propriedade na query, faz sentido ela estar tipada como obrigatória na usa interface?

```suggestion
  __typename?: string;
```

Reply:
Faz sentido